cg/test/management/stats.api.spec.js

```diff
@@ -41,6 +41,25 @@ beforeEach(() => {
 });
 afterAll(closeResources);
 
+async function clearEvents() {
+  try {
+    await deployContract();
+  } catch (e) {
+    winston.error(`Failed deploying contract ${e.toString()}`);
+  }
+}
+
+const SUBJECTS = [sha3('subject 1'), sha3('subject 2'), sha3('subject 3')];
+
+async function prepareEvents() {
+  await clearEvents();
+  let r;
+  for (let i in SUBJECTS) {
+    r = await recordConsentGivenTo(SUBJECTS[i], []);
+  }
+  return r.blockNumber;
+}
+
 describe('Stats endpoint', () => {
   it('Should display 0 stats properly if there is no entries yet', async () => {
```

Comment:
there are no entries, fix this too please

Reply:
I did not review existed tests. Ok, will fix this, no problem.

Reply:
Thx